aten/src/ATen/native/quantized/cpu/qnnpack_utils.h

```diff
@@ -114,11 +118,13 @@ struct PackedConvWeightsQnnp : public ConvPackedParamsBase<kSpatialDim> {
             {(uint32_t)dilation_[1], (uint32_t)dilation_[0]},
             {(uint32_t)padding_[0], (uint32_t)padding_[1],
              (uint32_t)padding_[0], (uint32_t)padding_[1]},
-            /*adjustment=*/{0, 0},
```

Comment:
what does "adjustment" mean here / is it ok to replace with output padding?

Reply:
Yea, the terminology sucks -- technically, this is adjustment to the output size, which is the same as the output_padding. The only difference is that adjustment refers to the size in general, while padding refers to the padding to every side separately.

Basically, I am using `asjustment` here because the qnnpack uses this as terminology, while pytorch uses "padding" -- should I change it?